src/clj/puppetlabs/services/jruby/jruby_puppet_schemas.clj

```diff
@@ -51,7 +51,11 @@
 
     * :http-client-cipher-suites - A list of legal SSL cipher suites that may
         be used when https client requests are made."
-  {:ruby-load-path              (schema/both [schema/Str] (schema/pred vector?))
+  ;; NOTE: there is a bug in the version of schema we're using, which causes
+  ;; the order of things that you put into a `both` to be very important.
+  ;; The `vector?` pred here MUST come before the `[schema/Str]`.  For more info
+  ;; see https://github.com/Prismatic/schema/issues/68
```

Comment:
I was chatting with Dan in clojurians yesterday, and he mentioned that when he tried upgrading to a newer version, it broke his recursive schemas.  We don't have any recursive schemas in Puppet Server yet, but we will as soon as we switch over to the new web routing library, so I was thinking I might try updating when I get back to working on that project.

If you want to create a ticket that'd be fine, but I'd add a comment capturing what Dan said.